be/src/util/json_util.cpp

```diff
@@ -62,4 +62,50 @@ std::string to_json(const Status& status) {
     return s.GetString();
 }
 
+std::string to_json(const std::map<std::string, std::map<std::string, std::string>>& value) {
+    rapidjson::StringBuffer buffer;
+    rapidjson::Writer writer(buffer);
+
+    writer.StartObject();
+
+    for (auto& iter : value) {
+        writer.Key(iter.first.c_str());
+        writer.StartObject();
+        for (auto innerIt = iter.second.begin(); innerIt != iter.second.end(); ++innerIt) {
+            writer.Key(innerIt->first.c_str());
+            writer.String(innerIt->second.c_str());
+        }
+        writer.EndObject();
+    }
+    writer.EndObject();
+    return buffer.GetString();
+}
+
+Status from_json(const std::string& json_value, std::map<std::string, std::map<std::string, std::string>>& map_result) {
```

Comment:
```suggestion
Status from_json(const std::string& json_value, std::map<std::string, std::map<std::string, std::string>>* map_result) {
```
prefer pointer as output argument